README.md

```diff
@@ -1 +1,9 @@
 Work for my participation in the Data for Science Residency Programme 2020.
+
```

Comment:
Just added some installation instructions for the project. I'm probably missing quite a few dependencies from the notebooks here!

Reply:
see spec-file.txt